index.html

```diff
@@ -171,7 +171,7 @@ <h2>About BIDS</h2>
         <p>A good introduction to the BIDS standard can be found in the <a href="http://www.nature.com/articles/sdata201644">paper published in Nature Scientific Data</a>.</p>
         <p>Look through some of our and collaborators <a href="https://osf.io/yn93h/">presentations on BIDS</a>.</p>
 	<p>Take a look at how the community <a href="https://medium.com/stanford-center-for-reproducible-neuroscience/bids-usage-survey-results-72637ff039c4">uses BIDS</a>.</p>
-	<p>Sign up to receive occasional <a href="https://docs.google.com/forms/d/1ZLi5qRTuX11KGK7qIidSdZvznFoXAqr2wh6003okv-0/edit">BIDS updates</a>.</p>
+	<p>Please sign up to receive occasional <a href="https://docs.google.com/forms/d/1ZLi5qRTuX11KGK7qIidSdZvznFoXAqr2wh6003okv-0/edit">BIDS updates</a> and <a href="https://twitter.com/BIDSstandard?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-show-count="false">Follow @BIDSstandard</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script></p>
```

Comment:
I would not ask with a "please" --> sounds too much like a request, while just "Sign up for ... " sounds more like an offer (despite the sentence being in imperative mode, gramatically)

Also we need to add that "follow BIDSstandard" means following on Twitter ... as opposed to some other social media :-)

```suggestion
	<p>Sign up to receive occasional <a href="https://docs.google.com/forms/d/1ZLi5qRTuX11KGK7qIidSdZvznFoXAqr2wh6003okv-0/edit">BIDS updates</a> and <a href="https://twitter.com/BIDSstandard?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-show-count="false">Follow @BIDSstandard on Twitter</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script></p>
```

Reply:
not sure why your rendering came up different (perhaps privacy/blocker extensions) but it has the twitter badge. so we do not need to say twitter

![Screen Shot 2019-07-15 at 4 45 06 PM](https://user-images.githubusercontent.com/35307458/61256029-02d7cd00-a720-11e9-993f-876128fd44a1.png)


```suggestion
	<p>Sign up to receive occasional <a href="https://docs.google.com/forms/d/1ZLi5qRTuX11KGK7qIidSdZvznFoXAqr2wh6003okv-0/edit">BIDS updates</a> and <a href="https://twitter.com/BIDSstandard?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-show-count="false">Follow @BIDSstandard</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script></p>
```